walkthroughs/ecs-service-connectivity/yelb/deployments/platformdeployment/AWS/ECS/deploy_db.sh

```diff
@@ -0,0 +1,21 @@
+export YELB_DB_ENDPOINT=$(aws cloudformation describe-stacks --stack-name yelb-fargate --query "Stacks[0].Outputs[?OutputKey=='YelbDBEndpointUrl'].OutputValue" --output text)
+export POSTGRES_PASSWORD=postgres_password
```

Comment:
Would it be possible to randomly generate this and inject it appropriately into the cloudformation template?

Reply:
the yelb application code is hard coded to use these values.